docs/dev-guide/contributing/branching.rst

```diff
@@ -61,12 +58,10 @@ Builds will be represented only as tags.
 Build Lifecycle
```

Comment:
This isn't part of your changes, so Github won't let me comment directly on it. 

Above this, in the "Build Tags" section, ``x.y`` needs to be enclosed by double-backticks